torchrl/envs/vec_env.py

```diff
@@ -1076,3 +1092,297 @@ def _run_worker_pipe_shared_mem(
             else:
                 # don't send env through pipe
                 child_pipe.send(("_".join([cmd, "done"]), None))
+
+
+class MultiThreadedEnvWrapper(_EnvWrapper):
+    """Wrapper for envpool-based multithreaded environments."""
+
+    _verbose: bool = False
+
+    def __init__(
+        self,
+        env: Optional["envpool.python.envpool.EnvPoolMixin"] = None,
+        **kwargs,
+    ):
+        if not _has_envpool:
+            raise ImportError(
```

Comment:
Another way is to keep the error in the try/except:
```
IMPORT_ERR_ENVPOOL = err
```

and then do `ImportError(...) from IMPORT_ERR_ENVPOOL`

Reply:
Makes sense, done